### ISSUE TITLE
joomla.json dowloads at meeting login submit

### ISSUE THREAD

**Issue by rmkeen** [jibon57/BigBlueButton_for_Joomla#11]
Jibon, first of all, nice work.
I am testing this out and found that when a user is logged into the Joomla site and logs into the meeting, the browser downloads joomla.json. If not logged into joomla, loging in to the meeting works as expected. I have the same results from event component login, meeting component login and the meeting module login, on both chrome and firefox.

**Comment by jibon57** [comment]
Try to disable `display_error` of PHP or from Joomla `Global Configuration` set `Error Reporting` to `none`.

**Comment by rmkeen** [comment]
I already had error reporting set to none.    	    	
      



On Fri, Apr 3, 2020 at 1:21 AM -0400, "Jibon L. Costa" <notifications@github.com> wrote:















Try to disable display_error of PHP or from Joomla Global Configuration set Error Reporting to none.

**Comment by jibon57** [comment]
Check your browser console to see if there any JavaScript conflict or errors

**Comment by rmkeen** [comment]
No error when the submit button is selected. Just opens save dialog box to save joomla.json when logged into Joomla. I did notice that the status label doesn’t appear either, but when not logged in the status label displays right away. 

 

I did find that a syntax error appears on page load of com_bigbluebutton eventview. There was a parenthesis in the name of the meeting room. I removed the parenthesis from the room name and refreshed the page. Now the redirection to the meeting room works whether logged into Joomla or not.
